applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py

```diff
@@ -118,7 +110,9 @@ def AssembleTestSuites():
     # For very long tests that should not be in nighly and you can use to validate
     validationSuite = suites['validation']
     validationSuite.addTest(BuoyancyTest('validationEulerian'))
+    validationSuite.addTest(AdjointVMSSensitivity2D('testCylinder'))
```

Comment:
Hi, would it be problematic if we leave at least this in the nighly suite. The issue in #8017 is about small suite ryt? At the moment we don't have any finite differencing full test in any CI (nighly or PR), I would like to have at least one if possible. :)

Reply:
There is still one (the element one). If I'm not wrong, the issue in #8017 is related with the all suite, which includes the small and nightly ones.

Reply:
Alryt :)... then its ok :)